cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md

```diff
@@ -894,6 +889,7 @@ the "breaking" tag. This ensures that the "Breaking" section of the release note
 description of what has broken from the past release. Label pull requests that contain deprecations
 with the "non-breaking" tag.
 
+(error-handling)=
 
 # Error Handling {#errors}
```

Comment:
is `{#errors}` still required?

Reply:
I think it's still allowing some of the links on GH to work. Having GH links work on a best-effort basis seems fine to me until we remove doxygen HTML.